apis/eventstream/index.yaml

```diff
@@ -34,12 +33,30 @@ get:
             type: string
             description: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#Event_stream_format
           examples:
-            head:
-              description: The node has finished processing, resulting in a new head
+            state:
+              description: |
+                The node has updated its state.  This event will be sent in three situations:
+
+                - a node has received a block, and updated state accordingly
+                - a node has _not_ received a block four seconds after the start of the slot, and updated state accordingly
+                - a node has carried out a chain reorganization, and updated state accordingly
+
+                The fields returned in the data for this event are:
+
+                  - **slot** the current slot as understood by the node
+                  - **block** the block root of the chain head as understood by the node
+                  - **state** the state root as understood by the node
+                  - **distance** the number of slots between the slot of the chain head and the current slot as understood by the node
+                  - **epoch_transition** _true_ if the slot marks the first slot in an epoch, otherwise _false_
+                  - **reorg** _true_ if the state has updated due to a reorganization, otherwise _false_
+                  - **reorg_distance** the number of slots between the slot of the block that caused a reorganization and the current slot as understood by the node
```

Comment:
This description of reorg_distance doesn't make sense to me - the reorg isn't necessarily caused by a new block being received, it may be because new attestations were received. So I'm a bit unclear what this value would actually be.

Validator clients need to determine the slot of the common ancestor block so this would either need to be that directly or the distance from the `slot` value back to the common ancestor slot.  That allows the client to invalidate only duties that are actually affected by the change instead of all of them.

Reply:
The idea of the reorg distance is, as you say, the distance from the current block back to the common ancestor slot.  "slot of the block that caused the a reorganization" tries to explain it, but obviously I'm not doing a great job there.  Happy to take suggestions on how to clarify this.

Reply:
I think the key term is common ancestor block so I'd probably describe it as something like:

```
   - **reorg_distance** the number of slots which were changed by this reorg. The common ancestor of the old and new forks is therefore at slot `slot - reorg_distance`.
```

Though it feels like it would be simpler to explain and use if it just included the ancestor slot directly.